crash-assistant breaks Divine Journey 2, should be excluded

### Describe the problem

When I try to run a server with the [Divine Journey 2](https://www.curseforge.com/minecraft/modpacks/divine-journey-2) modpack, I get a crash on launch that appears to be due to crash-assistant. Unsurprisingly, it resolves itself if I add `CF_EXCLUDE_MODS: crash-assistant` to the environment, but ideally it should be added to the globalExcludes list.

### Container definition

```
services:
  minecraft:
    image: itzg/minecraft-server:2025.12.0-java8@sha256:beb2cc1a665d93fbb72c60120c9f0751e94039cd37c4d975ce70d8abf18abb78
    container_name: minecraft-dj2-minimum-working-example
    ports:
      - 25580:25565
    environment:
      EULA: "TRUE"
      TYPE: AUTO_CURSEFORGE
      CF_API_KEY: ${CF_API_KEY:?error}
      CF_PAGE_URL: https://www.curseforge.com/minecraft/modpacks/divine-journey-2/files/7329323
      CF_FORCE_INCLUDE_MODS: hwyla
      CF_FORCE_SYNCHRONIZE: true
      SYNC_SKIP_NEWER_IN_DESTINATION: false

      WHITELIST: ${WHITELIST}
      OPS: ${OPS}
      MEMORY: 8G
      MAX_TICK_TIME: 600000

    tty: true
    stdin_open: true
    restart: no
```


### Container logs

```
[mc-image-helper] 02:19:04.878 INFO  : Mod file ./mods/mysticallib-1.12.2-1.13.0.jar already exists
[mc-image-helper] 02:19:04.878 INFO  : Mod file ./mods/Modular Diversity-1.8.jar already exists
[mc-image-helper] 02:19:04.878 INFO  : Mod file ./mods/TipTheScales-1.12.2-1.0.4.jar already exists
[mc-image-helper] 02:19:04.878 INFO  : Mod file ./mods/tweakersconstruct-1.12.2-1.6.1.jar already exists
[mc-image-helper] 02:19:04.879 INFO  : Mod file ./mods/mysticalworld-1.12.2-1.11.0.jar already exists
[mc-image-helper] 02:19:04.879 INFO  : Mod file ./mods/baubley-heart-canisters-1.12.2-1.2.3.jar already exists
[mc-image-helper] 02:19:04.879 INFO  : Mod file ./mods/LagGoggles-1.12.2-5.9-140.jar already exists
[mc-image-helper] 02:19:04.879 INFO  : Mod file ./mods/Placebo-1.12.2-1.6.1.jar already exists
[mc-image-helper] 02:19:04.880 INFO  : Mod file ./mods/MJRLegendsLib-1.12.2-1.2.1.jar already exists
[mc-image-helper] 02:19:04.880 INFO  : Excluding mod file 'Thaumic JEI' (thaumic-jei) due to configuration
[mc-image-helper] 02:19:04.880 INFO  : Mod file ./mods/bewitchment-1.12.2-0.0.22.65.jar already exists
[mc-image-helper] 02:19:04.880 INFO  : Mod file ./mods/GalacticraftTweaker-1.12.2-1.0.3.jar already exists
[mc-image-helper] 02:19:04.880 INFO  : Mod file ./mods/mystagradcompat-1.2.jar already exists
[mc-image-helper] 02:19:04.880 INFO  : Mod file ./mods/PlanetProgression-1.12.2-0.4.8.jar already exists
[mc-image-helper] 02:19:04.880 INFO  : Mod file ./mods/BasicNetherOres-1.12.2-1.0.5.0.jar already exists
[mc-image-helper] 02:19:04.881 INFO  : Mod file ./mods/tinkersjei-1.2.jar already exists
[mc-image-helper] 02:19:04.881 INFO  : Mod file ./mods/JustEnoughReactors-1.12.2-1.1.3.61.jar already exists
[mc-image-helper] 02:19:04.881 INFO  : Mod file ./mods/UndergroundBiomesConstructs-1.12-1.3.14.jar already exists
[mc-image-helper] 02:19:04.881 INFO  : Mod file ./mods/alib-1.0.12.jar already exists
[mc-image-helper] 02:19:04.881 INFO  : Mod file ./mods/alchemistry-1.12.2-42.jar already exists
[mc-image-helper] 02:19:04.882 INFO  : Mod file ./mods/BuildingGadgets-2.8.4.jar already exists
[mc-image-helper] 02:19:04.883 INFO  : Mod file ./mods/matc-1.0.1-hotfix.jar already exists
[mc-image-helper] 02:19:04.884 INFO  : Mod file ./mods/simpletrophies-1.2.2.1.jar already exists
[mc-image-helper] 02:19:04.885 INFO  : Mod file ./mods/jetif-1.12.2-1.5.2.jar already exists
[mc-image-helper] 02:19:04.885 INFO  : Mod file ./mods/demagnetize-1.12.2-1.1.2.jar already exists
[mc-image-helper] 02:19:04.885 INFO  : Mod file ./mods/WirelessFluidTerminal-1.12.2-1.0.4.jar already exists
[mc-image-helper] 02:19:04.885 INFO  : Mod file ./mods/AE2WTLib-1.12.2-1.0.34.jar already exists
[mc-image-helper] 02:19:04.886 INFO  : Mod file ./mods/EnderIO-endergy-1.12.2-5.3.72.jar already exists
[mc-image-helper] 02:19:04.886 INFO  : Mod file ./mods/ProjectIntelligence-1.12.2-1.0.9.28-universal.jar already exists
[mc-image-helper] 02:19:04.886 INFO  : Excluding mod file 'Tips' (tips) due to configuration
[mc-image-helper] 02:19:04.887 INFO  : Mod file ./mods/Patchouli-1.0-23.6.jar already exists
[mc-image-helper] 02:19:04.887 INFO  : Mod file ./mods/PackagedAuto-1.12.2-1.0.5.19.jar already exists
[mc-image-helper] 02:19:04.887 INFO  : Mod file ./mods/tmel-1.12.2-1.4.0.0.jar already exists
[mc-image-helper] 02:19:04.888 INFO  : Mod file ./mods/betternether-0.1.8.6.jar already exists
[mc-image-helper] 02:19:04.888 INFO  : Mod file ./mods/EnderIO-conduits-mekanism-1.12.2-5.3.72.jar already exists
[mc-image-helper] 02:19:04.888 INFO  : Mod file ./mods/simplybackpacks-1.4.9.jar already exists
[mc-image-helper] 02:19:04.889 INFO  : Mod file ./mods/inworldcrafting-1.12.2-1.2.0-universal.jar already exists
[mc-image-helper] 02:19:04.889 INFO  : Mod file ./mods/EnderTweaker-1.12.2-1.2.3.jar already exists
[mc-image-helper] 02:19:04.889 INFO  : Mod file ./mods/Wasaila-1.0.jar already exists
[mc-image-helper] 02:19:04.889 INFO  : Mod file ./mods/Draconic-Additions-1.12.2-1.15.1.39-universal.jar already exists
[mc-image-helper] 02:19:04.889 INFO  : Mod file ./mods/FTBBackups-1.1.0.1.jar already exists
[mc-image-helper] 02:19:04.890 INFO  : Mod file ./mods/knowledgeshare-1.1.jar already exists
[mc-image-helper] 02:19:04.890 INFO  : Mod file ./mods/ThaumicAugmentation-1.12.2-2.1.14.jar already exists
[mc-image-helper] 02:19:04.890 INFO  : Mod file ./mods/PlaneFix-1.12.2-1.0.0.jar already exists
[mc-image-helper] 02:19:04.890 INFO  : Mod file ./mods/AE-Net-Vis-Tool-1.12.2-1.0.3.0-universal.jar already exists
[mc-image-helper] 02:19:04.892 INFO  : Mod file ./mods/modularmagic-1.4.0.jar already exists
[mc-image-helper] 02:19:04.893 INFO  : Mod file ./mods/MoreTweaker-1.0.19.jar already exists
[mc-image-helper] 02:19:04.894 INFO  : Mod file ./mods/Requious Frakto-0.12.jar already exists
[mc-image-helper] 02:19:04.894 INFO  : Mod file ./mods/bettercaves-1.12.2-2.0.4.jar already exists
[mc-image-helper] 02:19:04.894 INFO  : Mod file ./mods/DivineRPG-1.7.1.jar already exists
[mc-image-helper] 02:19:04.895 INFO  : Mod file ./mods/supersoundmuffler-revived_1.12.2_1.0.2.10.jar already exists
[mc-image-helper] 02:19:04.895 INFO  : Mod file ./mods/plustic-8.0.3.jar already exists
[mc-image-helper] 02:19:04.895 INFO  : Mod file ./mods/framedcompactdrawers-1.2.7.jar already exists
[mc-image-helper] 02:19:04.896 INFO  : Mod file ./mods/TickCentral-3.2.jar already exists
[mc-image-helper] 02:19:04.896 INFO  : Mod file ./mods/spark-forge.jar already exists
[mc-image-helper] 02:19:04.897 INFO  : Mod file ./mods/integratednbt-1.2.2.jar already exists
[mc-image-helper] 02:19:04.897 INFO  : Mod file ./mods/extendedcrafting-nomifactory-edition-1.7.0.6.jar already exists
[mc-image-helper] 02:19:04.897 INFO  : Mod file ./mods/zenutils-1.17.1.jar already exists
[mc-image-helper] 02:19:04.898 INFO  : Mod file ./mods/extracpus-1.12.2-1.2.1.jar already exists
[mc-image-helper] 02:19:04.898 INFO  : Mod file ./mods/QuarkRotN-r1.6-191.jar already exists
[mc-image-helper] 02:19:04.899 INFO  : Mod file ./mods/tinkersaether-1.4.1.jar already exists
[mc-image-helper] 02:19:04.899 INFO  : Mod file ./mods/!mixinbooter-10.7.jar already exists
[mc-image-helper] 02:19:04.900 INFO  : Mod file ./mods/Forgelin-Continuous-2.2.21.0.jar already exists
[mc-image-helper] 02:19:04.902 INFO  : Mod file ./mods/astral-level-nerf-1.0.1-all.jar already exists
[mc-image-helper] 02:19:04.902 INFO  : Mod file ./mods/uberconduitprobe-1.0.3.jar already exists
[mc-image-helper] 02:19:04.903 INFO  : Mod file ./mods/ExtraCells-1.12.2-2.6.7.jar already exists
[mc-image-helper] 02:19:04.906 INFO  : Mod file ./mods/oauth-1.06.4.jar already exists
[mc-image-helper] 02:19:04.907 INFO  : Mod file ./mods/Galacticraft-1.12.2-4.0.7.jar already exists
[mc-image-helper] 02:19:04.908 INFO  : Mod file ./mods/ae2-uel-v0.56.4.jar already exists
[mc-image-helper] 02:19:04.909 INFO  : Mod file ./mods/dj2addons-1.2.1.1.1.jar already exists
[mc-image-helper] 02:19:04.910 INFO  : Mod file ./mods/UniversalTweaks-1.12.2-1.17.0.jar already exists
[mc-image-helper] 02:19:04.910 INFO  : Mod file ./mods/thaumicenergistics-extended_life-2.2.7.jar already exists
[mc-image-helper] 02:19:04.913 INFO  : Mod file ./mods/!configanytime-3.0.jar already exists
[mc-image-helper] 02:19:04.913 INFO  : Mod file ./mods/vintagefix-0.6.2.jar already exists
[mc-image-helper] 02:19:04.914 INFO  : Mod file ./mods/!Red-Core-MC-1.8-1.12-0.6.jar already exists
[mc-image-helper] 02:19:04.915 INFO  : Mod file ./mods/betterp2p-1.12.2-1.2.3-extended_life.jar already exists
[mc-image-helper] 02:19:04.915 INFO  : Mod file ./mods/Alfheim-1.6.jar already exists
[mc-image-helper] 02:19:04.915 INFO  : Mod file ./mods/serializationisbad-1.5.2.jar already exists
[mc-image-helper] 02:19:04.915 INFO  : Mod file ./mods/omniwand-1.12.2-2.0.1.jar already exists
[mc-image-helper] 02:19:04.915 INFO  : Mod file ./mods/`FermiumBooter-1.4.1.jar already exists
[mc-image-helper] 02:19:04.916 INFO  : Mod file ./mods/fermiumasm-5.28.jar already exists
[mc-image-helper] 02:19:04.916 INFO  : Mod file ./mods/twilightforest-1.12.2-3.14.jar already exists
[mc-image-helper] 02:19:04.916 INFO  : Mod file ./mods/integratedderivative-1.1.2.jar already exists
[mc-image-helper] 02:19:04.917 INFO  : Mod file ./mods/ThaumcraftFix-1.12.2-1.1.8.jar already exists
[mc-image-helper] 02:19:04.918 INFO  : Mod file ./mods/natura-legacy-1.12.2-4.4.0.jar already exists
[mc-image-helper] 02:19:04.920 INFO  : Mod file ./mods/NoiseThreader-1.1.2.jar already exists
[mc-image-helper] 02:19:04.921 INFO  : Mod file ./mods/Botania r1.10-369b.jar already exists
[mc-image-helper] 02:19:04.922 INFO  : Mod file ./mods/endermodpacktweaks-0.5.11.jar already exists
[mc-image-helper] 02:19:05.027 INFO  : Downloaded mod file mods/Hwyla-1.8.26-B41_1.12.2.jar
[mc-image-helper] 02:19:05.152 INFO  : Downloaded mod file mods/!!!CrashAssistant-forge-1.12.2-1.10.20.jar
[mc-image-helper] 02:19:05.368 INFO  : Forge version 14.23.5.2860 for minecraft version 1.12.2 is already installed
[init] Copying any mods from /mods to /data/mods
[init] Copying any configs from /config to /data/config
[init] Enabling whitelist functionality
[mc-image-helper] 02:19:07.234 INFO  : Created/updated 1 property in /data/server.properties
[init] Setting initial memory to 8G and max to 8G
[init] Starting the Minecraft server...
[Log4jPatcher] [INFO] Transforming org/apache/logging/log4j/core/lookup/JndiLookup
[Log4jPatcher] [INFO] Transforming org/apache/logging/log4j/core/pattern/MessagePatternConverter
[Log4jPatcher] [WARN]  Unable to find noLookups:Z field in org/apache/logging/log4j/core/pattern/MessagePatternConverter
[02:19:10] [main/INFO] [LaunchWrapper]: Loading tweak class name net.minecraftforge.fml.common.launcher.FMLServerTweaker
[02:19:10] [main/INFO] [LaunchWrapper]: Using primary tweak class name net.minecraftforge.fml.common.launcher.FMLServerTweaker
[02:19:10] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.common.launcher.FMLServerTweaker
[02:19:10] [main/INFO] [FML]: Forge Mod Loader version 14.23.5.2860 for Minecraft 1.12.2 loading
[02:19:10] [main/INFO] [FML]: Java is OpenJDK 64-Bit Server VM, version 1.8.0_312, running on Linux:amd64:6.17.0-8-generic, installed at /opt/java/openjdk
[02:19:10] [main/INFO] [FML]: Searching /data/./mods for mods
[02:19:10] [main/INFO] [FML]: Searching /data/./mods/1.12.2 for mods
[02:19:10] [main/WARN] [FML]: Found FMLCorePluginContainsFMLMod marker in !!!CrashAssistant-forge-1.12.2-1.10.20.jar. This is not recommended, @Mods should be in a separate jar from the coremod.
[02:19:10] [main/WARN] [FML]: The coremod CrashAssistantEntrypoint (dev.kostromdan.mods.crash_assistant.forge_coremod.CrashAssistantEntrypoint) is not signed!
[02:19:10] [main/ERROR] [LaunchWrapper]: Unable to launch
java.lang.RuntimeException: An error occurred trying to configure the Minecraft home at /data/. for Forge Mod Loader
	at net.minecraftforge.fml.relauncher.FMLLaunchHandler.setupHome(FMLLaunchHandler.java:111) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.relauncher.FMLLaunchHandler.setupServer(FMLLaunchHandler.java:87) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.relauncher.FMLLaunchHandler.configureForServerLaunch(FMLLaunchHandler.java:47) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.common.launcher.FMLServerTweaker.injectIntoClassLoader(FMLServerTweaker.java:62) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:115) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_312]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_312]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_312]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_312]
	at net.minecraftforge.fml.relauncher.ServerLaunchWrapper.run(ServerLaunchWrapper.java:70) [forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.relauncher.ServerLaunchWrapper.main(ServerLaunchWrapper.java:34) [forge-1.12.2-14.23.5.2860.jar:?]
Caused by: java.lang.NoClassDefFoundError: dev/kostromdan/mods/crash_assistant/common_config/utils/ProcessHandleLinuxImpl$LinuxCLibrary
	at dev.kostromdan.mods.crash_assistant.common_config.utils.ProcessHandleLinuxImpl.<clinit>(ProcessHandleLinuxImpl.java:117) ~[ProcessHandleLinuxImpl.class:?]
	at dev.kostromdan.mods.crash_assistant.common_config.utils.ProcessHelper.<clinit>(ProcessHelper.java:33) ~[ProcessHelper.class:?]
	at dev.kostromdan.mods.crash_assistant.common_config.platform.PlatformHelp.<clinit>(PlatformHelp.java:31) ~[PlatformHelp.class:?]
	at dev.kostromdan.mods.crash_assistant.forge_coremod.CrashAssistantEntrypoint.<init>(CrashAssistantEntrypoint.java:23) ~[!!!CrashAssistant-forge-1.12.2-1.10.20.jar:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_312]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_312]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_312]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_312]
	at java.lang.Class.newInstance(Class.java:442) ~[?:1.8.0_312]
	at net.minecraftforge.fml.relauncher.CoreModManager.loadCoreMod(CoreModManager.java:594) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.relauncher.CoreModManager.discoverCoreMods(CoreModManager.java:451) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.relauncher.CoreModManager.handleLaunch(CoreModManager.java:265) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.relauncher.FMLLaunchHandler.setupHome(FMLLaunchHandler.java:107) ~[forge-1.12.2-14.23.5.2860.jar:?]
	... 11 more
Caused by: java.lang.ClassNotFoundException: dev.kostromdan.mods.crash_assistant.common_config.utils.ProcessHandleLinuxImpl$LinuxCLibrary
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_312]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_312]
	at dev.kostromdan.mods.crash_assistant.common_config.utils.ProcessHandleLinuxImpl.<clinit>(ProcessHandleLinuxImpl.java:117) ~[ProcessHandleLinuxImpl.class:?]
	at dev.kostromdan.mods.crash_assistant.common_config.utils.ProcessHelper.<clinit>(ProcessHelper.java:33) ~[ProcessHelper.class:?]
	at dev.kostromdan.mods.crash_assistant.common_config.platform.PlatformHelp.<clinit>(PlatformHelp.java:31) ~[PlatformHelp.class:?]
	at dev.kostromdan.mods.crash_assistant.forge_coremod.CrashAssistantEntrypoint.<init>(CrashAssistantEntrypoint.java:23) ~[!!!CrashAssistant-forge-1.12.2-1.10.20.jar:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_312]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_312]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_312]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_312]
	at java.lang.Class.newInstance(Class.java:442) ~[?:1.8.0_312]
	at net.minecraftforge.fml.relauncher.CoreModManager.loadCoreMod(CoreModManager.java:594) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.relauncher.CoreModManager.discoverCoreMods(CoreModManager.java:451) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.relauncher.CoreModManager.handleLaunch(CoreModManager.java:265) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.relauncher.FMLLaunchHandler.setupHome(FMLLaunchHandler.java:107) ~[forge-1.12.2-14.23.5.2860.jar:?]
	... 11 more
Caused by: java.lang.NoClassDefFoundError: com/sun/jna/Library
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:1.8.0_312]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:756) ~[?:1.8.0_312]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142) ~[?:1.8.0_312]
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:182) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_312]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_312]
	at dev.kostromdan.mods.crash_assistant.common_config.utils.ProcessHandleLinuxImpl.<clinit>(ProcessHandleLinuxImpl.java:117) ~[ProcessHandleLinuxImpl.class:?]
	at dev.kostromdan.mods.crash_assistant.common_config.utils.ProcessHelper.<clinit>(ProcessHelper.java:33) ~[ProcessHelper.class:?]
	at dev.kostromdan.mods.crash_assistant.common_config.platform.PlatformHelp.<clinit>(PlatformHelp.java:31) ~[PlatformHelp.class:?]
	at dev.kostromdan.mods.crash_assistant.forge_coremod.CrashAssistantEntrypoint.<init>(CrashAssistantEntrypoint.java:23) ~[!!!CrashAssistant-forge-1.12.2-1.10.20.jar:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_312]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_312]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_312]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_312]
	at java.lang.Class.newInstance(Class.java:442) ~[?:1.8.0_312]
	at net.minecraftforge.fml.relauncher.CoreModManager.loadCoreMod(CoreModManager.java:594) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.relauncher.CoreModManager.discoverCoreMods(CoreModManager.java:451) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.relauncher.CoreModManager.handleLaunch(CoreModManager.java:265) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.relauncher.FMLLaunchHandler.setupHome(FMLLaunchHandler.java:107) ~[forge-1.12.2-14.23.5.2860.jar:?]
	... 11 more
Caused by: java.lang.ClassNotFoundException: com.sun.jna.Library
	at java.net.URLClassLoader.findClass(URLClassLoader.java:387) ~[?:1.8.0_312]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_312]
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352) ~[?:1.8.0_312]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_312]
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:106) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_312]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_312]
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:1.8.0_312]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:756) ~[?:1.8.0_312]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142) ~[?:1.8.0_312]
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:182) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_312]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_312]
	at dev.kostromdan.mods.crash_assistant.common_config.utils.ProcessHandleLinuxImpl.<clinit>(ProcessHandleLinuxImpl.java:117) ~[ProcessHandleLinuxImpl.class:?]
	at dev.kostromdan.mods.crash_assistant.common_config.utils.ProcessHelper.<clinit>(ProcessHelper.java:33) ~[ProcessHelper.class:?]
	at dev.kostromdan.mods.crash_assistant.common_config.platform.PlatformHelp.<clinit>(PlatformHelp.java:31) ~[PlatformHelp.class:?]
	at dev.kostromdan.mods.crash_assistant.forge_coremod.CrashAssistantEntrypoint.<init>(CrashAssistantEntrypoint.java:23) ~[!!!CrashAssistant-forge-1.12.2-1.10.20.jar:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_312]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_312]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_312]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_312]
	at java.lang.Class.newInstance(Class.java:442) ~[?:1.8.0_312]
	at net.minecraftforge.fml.relauncher.CoreModManager.loadCoreMod(CoreModManager.java:594) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.relauncher.CoreModManager.discoverCoreMods(CoreModManager.java:451) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.relauncher.CoreModManager.handleLaunch(CoreModManager.java:265) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.relauncher.FMLLaunchHandler.setupHome(FMLLaunchHandler.java:107) ~[forge-1.12.2-14.23.5.2860.jar:?]
	... 11 more
A problem occurred running the Server launcher.[02:19:10] [main/INFO] [STDERR]: [net.minecraftforge.fml.relauncher.ServerLaunchWrapper:main:34]: java.lang.reflect.InvocationTargetException
[02:19:10] [main/INFO] [STDERR]: [net.minecraftforge.fml.relauncher.ServerLaunchWrapper:main:34]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[02:19:10] [main/INFO] [STDERR]: [net.minecraftforge.fml.relauncher.ServerLaunchWrapper:main:34]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[02:19:10] [main/INFO] [STDERR]: [net.minecraftforge.fml.relauncher.ServerLaunchWrapper:main:34]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[02:19:10] [main/INFO] [STDERR]: [net.minecraftforge.fml.relauncher.ServerLaunchWrapper:main:34]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[02:19:10] [main/INFO] [STDERR]: [net.minecraftforge.fml.relauncher.ServerLaunchWrapper:main:34]: 	at net.minecraftforge.fml.relauncher.ServerLaunchWrapper.run(ServerLaunchWrapper.java:70)
[02:19:10] [main/INFO] [STDERR]: [net.minecraftforge.fml.relauncher.ServerLaunchWrapper:main:34]: 	at net.minecraftforge.fml.relauncher.ServerLaunchWrapper.main(ServerLaunchWrapper.java:34)
[02:19:10] [main/INFO] [STDERR]: [net.minecraftforge.fml.relauncher.ServerLaunchWrapper:run:75]: Caused by: net.minecraftforge.fml.relauncher.FMLSecurityManager$ExitTrappedException
[02:19:10] [main/INFO] [STDERR]: [net.minecraftforge.fml.relauncher.ServerLaunchWrapper:run:75]: 	at net.minecraftforge.fml.relauncher.FMLSecurityManager.checkPermission(FMLSecurityManager.java:49)
[02:19:10] [main/INFO] [STDERR]: [net.minecraftforge.fml.relauncher.ServerLaunchWrapper:run:75]: 	at java.lang.SecurityManager.checkExit(SecurityManager.java:761)
[02:19:10] [main/INFO] [STDERR]: [net.minecraftforge.fml.relauncher.ServerLaunchWrapper:run:75]: 	at java.lang.Runtime.exit(Runtime.java:107)
[02:19:10] [main/INFO] [STDERR]: [net.minecraftforge.fml.relauncher.ServerLaunchWrapper:run:75]: 	at java.lang.System.exit(System.java:973)
[02:19:10] [main/INFO] [STDERR]: [net.minecraftforge.fml.relauncher.ServerLaunchWrapper:run:75]: 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:138)
[02:19:10] [main/INFO] [STDERR]: [net.minecraftforge.fml.relauncher.ServerLaunchWrapper:run:75]: 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
[02:19:10] [main/INFO] [STDERR]: [net.minecraftforge.fml.relauncher.ServerLaunchWrapper:run:75]: 	... 6 more
2025-12-28T02:19:10.604Z	WARN	mc-server-runner	Minecraft server failed. Inspect logs above for errors that indicate cause. DO NOT report this line as an error.	{"exitCode": 1}
2025-12-28T02:19:10.612Z	INFO	mc-server-runner	Done
Container stopped
```

--- a/files/cf-exclude-include.json
+++ b/files/cf-exclude-include.json
@@ -35,6 +35,7 @@
     "controllable",
     "controlling",
     "craftpresence",
+    "crash-assistant",
     "cull-less-leaves",
     "ctm",
     "custom-main-menu",

--- a/files/modrinth-exclude-include.json
+++ b/files/modrinth-exclude-include.json
@@ -32,6 +32,7 @@
     "continuity",
     "controlling",
     "craftpresence",
+    "CrashAssistant",
     "Cull Less Leaves",
     "cwb",
     "DisableCustomWorldsAdvice",

